spec/system/order_filter_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Order filtering', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  context 'when an employee filters orders' do
+    let!(:employee) { create(:employee) }
+    let!(:delivered_order) { create(:order, status: 'delivered') }
+    let!(:not_delivered_order) { create(:order, status: 'not_delivered') }
+
+    it 'Filtering by delivered, not_delivered and all' do
```

Comment:
suggestion only - check out betterspec it's a convention of naming the specs

```suggestion
    it 'filters by the delivery status' do
```

Reply:
Of course I will read about it